### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by Munsa1**
In this milestone we added:
- luxon logic for adding the current time on the page
- added a navbar with 3 clickable elements (booklist, create new, about us)
- "book list" is displaying all the books added to the local storage
- "create new" contains the form for adding new books
- "about us" contains the contact information

### CODE REVIEW

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @Munsa1 and @igorkol91 👋🏾 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a PR description. Kindly add a brief summary of what you achieved in the description box.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -10,3 +10,9 @@
   border-radius: 5px;
   background-color: rgb(230, 230, 230);
 }
+
+.info {
+  list-style: circle;
+  margin-top: 15px;
+  text-align: left;
+}
```
Comment (Lameck1) [inline style.css]: - [x] The CSS file does not have enough style to match up with the minimum style requirement as in the provided wireframe. We are given Wireframes to be guiding principles as to how the UI should result. So, we've not done justice here as far as this issue is concerned. [Please watch this video recording in which I address this issue.](https://www.loom.com/share/54bca187b89249ad9029c347f0a6f275). </br> Finally, a screenshot of the wireframes 👇🏼. Please let our pages look at least like what we are seeing below.

| The Wireframes |
| --- |
| wireframe |

File: index.html [hunk 2]
```diff
@@ -8,17 +8,22 @@
...
-    </form>
+<body  class="text-center">
+    <div class="d-flex list-unstyled justify-content-between">
+        <h4 class="mx-5">Awesome books</h4>
+        <ul class="d-flex list-unstyled justify-content-end mx-5">
+            <li class="mx-3"><a id="main-page" class="text-decoration-none" href="#">Book List</a></li>
+            <li class="mx-3"><a id="create-new-page" class="text-decoration-none" href="#">Create New</a></li>
+            <li class="mx-3"><a id="about-page" class="text-decoration-none" href="#">About Us</a></li>
+        </ul>
+    </div>
```
Comment (Lameck1) [inline index.html]: - [x] [OPTIONAL] I suggest using more semantic HTML tags to communicate the real intentions of a given section of an HTML document. Take a look at [THIS](https://www.w3schools.com/html/html5_semantic_elements.asp) to learn more about semantic HTML. Contents required in this section are included though 👍🏼.

File: index.html [hunk 3]
```diff
@@ -8,17 +8,22 @@
...
+    <div class="d-flex justify-content-end mx-5">
+        <p id="time-p"></p>
+    </div>
+    <h1>All Awesome Books</h1>
+    <main  class="d-flex align-items-center flex-column">
+  </main>
     <script src="./main.js" type="module"></script>
+    <script src="./luxon.js"></script>
 </body>
```
Comment (Lameck1) [inline index.html:31]: - [x] We have forgotten the footer here. The wireframe requires this of us. Please let's work on it.

File: main.js [hunk 4]
```diff
@@ -26,18 +38,88 @@ const refreshDOM = () => {
...
   });
 };
 window.onload = refreshDOM;
-createNew.addEventListener('click', (e) => {
-  e.preventDefault();
-  add.add();
-  bookContainer.innerHTML = '';
+window.onload = () => {
+  time.innerHTML = `${now.c.month} ${now.c.day} ${now.c.year}, ${now.c.hour}:${now.c.minute}:${now.c.second}`;
+};
```
Comment (Lameck1) [inline main.js]: - [x] This is the problem that makes your page not load books when the page is loaded for the first time or when refreshed. The second call to `window.load` overrides/overshadows the first. I suggest you have one instance of `window.load` and do everything else you wanted with `window.load`. Somethign like this 👇🏼:

```JavaScript
window.onload = () => {
  refreshDOM();
  time.innerHTML = `${ now.c.month } ${ now.c.day } ${ now.c.year }, ${ now.c.hour }:${ now.c.minute }:${ now.c.second }`;
};
```

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi @igorkol91 & @Munsa1 👋🏼, 

Good job so far! You've come a long way with this project and your good effort is recognizable.

In this last milestone, we would like to ship a good product if not a perfect one. For this reason, there are suggestions that we need to look into just before we could move ahead. See below 👇🏼:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (DeliceLydia) [reply style.css]: Addressed this well!

File: index.html (hunk 3 above)
Comment (DeliceLydia) [reply index.html:31]: Well addressed this

File: main.js (hunk 4 above)
Comment (DeliceLydia) [reply main.js]: resolved it!

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**